tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/balo/imports/ImportsTests.java

```diff
@@ -43,42 +39,21 @@
 
     private static final String USER_HOME = "user.home";
     private String previousUserHome;
-    private Path tempBir;
     private Path imports;
-    private Path homeBirCache;
-    private Path birDir;
-    private List<Path> birFiles;
 
     @BeforeClass(description = "Build the necessary modules.")
     public void setup() throws IOException {
         Path tempDir = Files.createTempDirectory("temp-bal-home");
-
-        tempBir = Paths.get(System.getProperty("user.dir"), "build", "test-bir-temp");
-
         imports = Paths.get("test-src", "balo", "imports");
-
-        homeBirCache = tempDir.resolve(Paths.get(".ballerina", "bir_cache" + "-" +
-                RepoUtils.getBallerinaVersion()));
-
         previousUserHome = System.getProperty(USER_HOME);
         System.setProperty(USER_HOME, tempDir.toString());
-
-        // Delete all existing bir files.
-        BFileUtil.deleteAll(tempBir, "testModule.bir");
     }
 
     @Test(description = "Get the version from the source file.")
-    public void testVersionSupportImportFromSourceFile() throws IOException {
+    public void testVersionSupportImportFromSourceFile() {
         BaloCreator.cleanCacheDirectories();
         BaloCreator.createAndSetupBalo(imports.resolve(Paths.get("test_module1", "test_module")).toString(),
                 "testOrg", "testModule", "1.1.0");
-        birFiles = Files.find(tempBir, Integer.MAX_VALUE, (path, attribute) ->
-                path.toString().contains("testModule.bir")
-                        && !path.toString().contains("imports")).collect(Collectors.toList());
-        birDir = Files.createDirectories(homeBirCache.resolve(Paths.get("testOrg", "testModule", "1.1.0")));
-        BFileUtil.copy(birFiles.get(0), birDir.resolve(Paths.get("testModule.bir")));
```

Comment:
Why we need these changes in this test?

Reply:
Because, in test, we don't write bir binaries anymore, with the new jBallerina java rewrite. So the above lines were not needed.